src/editors/sharedComponents/TinyMceWidget/hooks.js

```diff
@@ -98,21 +98,28 @@ export const replaceStaticWithAsset = ({
       const assetName = parseAssetName(src);
       const displayName = isStatic ? staticName : assetName;
       const isCorrectAssetFormat = assetSrc.startsWith('/asset') && assetSrc.match(/\/asset-v1:\S+[+]\S+[@]\S+[+]\S+[@]/g)?.length >= 1;
```

Comment:
[mega-nit] you could move this variable declaration into the `if (!isLibraryKey...)` block, as the value of `isCorrectAssetFormat` is only reasonable if we know that we're looking at a modulestore course asset.